Content.Server/Fax/FaxableObjectComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Paper;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Fax;
+
+[RegisterComponent]
+public sealed partial class FaxableObjectComponent : Component
+{
+    /// <summary>
+    /// Name with which the fax will be visible to others on the network
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField]
+    public string Identity = "paper";
+}
```

Comment:
Don't use flaky string checks for faxecution, this is how you get https://arstechnica.com/gaming/2018/07/a-years-old-one-letter-typo-led-to-aliens-colonial-marines-awful-ai/

Reply:
is it ok to use a string for the sprite state as in inserting_hamster. If no string is inserted it defaults to paper sprite and if its incorrectly spelt it will just cause an ERROR sprite so no major issues with it.